pontoon/contributors/utils.py

```diff
@@ -104,6 +104,9 @@ def users_with_translations_counts(
     # Assign properties to user objects.
     contributors = User.objects.filter(pk__in=user_stats.keys())
 
+    # Exclude system users
+    contributors = contributors.exclude(profile__system_user=True)
+
     # Exclude deleted users.
     contributors = contributors.filter(is_active=True)
```

Comment:
To be consistent with the line you added above, please invert this QuerySet to use `exclude()` instead of `filter()` and filp the boolean.

Nit: Please also make sure punctuation in the comments is consistent. :)